src/common.pgo.runtime.props

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- From our friends at MUX: https://github.com/microsoft/microsoft-ui-xaml/blob/main/PGO.runtime.props -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <PlatformShortName Condition="'$(PlatformShortName)'==''">$(Platform)</PlatformShortName>
+        <PlatformShortName Condition="'$(PlatformShortName)'=='Win32'">x86</PlatformShortName>
+        <PlatformShortName Condition="'$(PlatformShortName)'=='AnyCPU' and '$(Prefer32Bit)'==''">x64</PlatformShortName>
+        <PlatformShortName Condition="'$(PlatformShortName)'=='AnyCPU' and '$(Prefer32Bit)'=='true'">x64</PlatformShortName>
+        <PlatformShortName Condition="'$(PlatformShortName)'=='AnyCPU' and '$(Prefer32Bit)'=='false'">x64</PlatformShortName>
+        <PGOFilesDir>$(VCToolsInstallDir)bin\Host$(PlatformShortName)\$(PlatformShortName)</PGOFilesDir>

```

Comment:
funny this is actually not the "correct" way to get the Host folder. The Host is determined by the flag that says "prefer native toolchain?". This will probably ride and be fine, however, but it will cause weirdness when arm64 is lit up (since we aren't using arm64-hosted tools)